easy_phi/tests/run_tests.py

```diff
@@ -6,6 +6,7 @@
 TEST_MODULES = [
     'easy_phi.tests.handlers_test',
     'easy_phi.tests.mod_conf_patch_test',
+    'easy_phi.tests.regression_test',
```

Comment:
I think it's a very good point, but it's nice-to-have automation. We can observe regression all the time, but during release I can record the regression in separate file.